cli/server.go

```diff
@@ -813,6 +814,21 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 				options.SwaggerEndpoint = cfg.Swagger.Enable.Value()
 			}
 
+			batchStatsTicker := time.NewTicker(29 * time.Second) // Hard-coding.
```

Comment:
review: Would this be better set at a lower interval, or made configurable?

Reply:
First of all, 29 sec looks magical :) You may need to comment on why is it exactly this.

I didn't look deeper into the review yet, but you may want to introduce 2 limits: timeout and buffer size are getting full.

Reply:
Made the interval 1 second instead by default.